types/libsodium-wrappers/index.d.ts

```diff
@@ -464,6 +464,8 @@ export function crypto_pwhash_str(password: string | Uint8Array, opsLimit: numbe
 
 export function crypto_pwhash_str_verify(hashed_password: string, password: string | Uint8Array): boolean;
 
+export function crypto_pwhash_str_needs_rehash(hashed_password: string, opsLimit: number, memLimit: number): boolean;
```

Comment:
Let's stick to camelCase :)

```suggestion
export function crypto_pwhash_str_needs_rehash(hashedPassword: string, opsLimit: number, memLimit: number): boolean;
```